- Fix path to /usr/local/share

- Don't fail if unrecognised autoconf command-line flags are present
(avoid problem with --disable-gtk-doc)

- Permit building if security/openpam is installed.

- Don't fail if pkg-config --list-all returns an error code.

- gnome-open -> xdg-open

Index: configure
--- configure.orig
+++ configure
@@ -2826,11 +2826,6 @@ ac_config_headers="$ac_config_headers config.h"
 echo "current directory: `pwd`"
 echo "command line was: $0 $@"
 
-if ! test -z "$ac_unrecognized_opts" ; then
-  echo "" >&2
-  exit 2
-fi
-
 ###############################################################################
 #
 #       Autoheader stuff
@@ -15029,7 +15024,7 @@ printf "%s\n" "$ac_cv_gtk_config_libs" >&6; }
 
   GTK_DATADIR=""
   if test "$have_gtk" = yes; then
-    GTK_DATADIR=`$pkg_config --variable=prefix gtk+-2.0`
+    GTK_DATADIR=`$pkg_config --variable=prefix gtk+-3.0`
     GTK_DATADIR="$GTK_DATADIR/share"
   fi
 
@@ -15060,9 +15055,9 @@ fi
 
 # Check for the various Gnome help and URL loading programs.
 #
-WITH_BROWSER=gnome-open
+WITH_BROWSER=xdg-open
 if test "$have_gtk" = yes; then
-  for ac_prog in gnome-open
+  for ac_prog in xdg-open
 do
   # Extract the first word of "$ac_prog", so it can be a program name with args.
 set dummy $ac_prog; ac_word=$2
@@ -21754,7 +21749,6 @@ else
     warn2 "The errors are:"
     echo ''
     echo "$pkgerr" | sed 's/^/             > /g'
-    CONF_STATUS=1
   fi
 fi
 
@@ -22043,7 +22037,6 @@ elif test "$have_pam" = no ; then
   if test -d /etc/pam.d -o -f /etc/pam.conf ; then
     warn  "Your system seems to have PAM, but PAM is not being used."
     warn2 "That is probably not going to work out well."
-    CONF_STATUS=1
   fi
 fi
 
